lib/chitter.rb

```diff
@@ -0,0 +1,20 @@
+require "pg"
+
+class Chitter
+  def self.all
+    if ENV["ENVIRONMENT"] == "test"
```

Comment:
It's worth considering that this if/else statement is repeated below as well, to follow the DRY principle, you could, for example, create a function that takes the `environment` information and then reuse it to establish the connection with the test or the main database.